src/client/pages/CountryHome/Collaborators/UserList/UserList.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
New component, this replaces the old UserList component and underlying is simply usage for `TablePaginated` components.